content/index.md

```diff
@@ -36,24 +36,35 @@ that are [reusable across multiple languages and frameworks](#portability).
 
 This `Card` component is [implemented as a JavaScript function](#components).
 
-
-Stateless HTML Rendering
-------------------------
+<div class="icon-heading">
+    <div class="badge">
+        <svg class="icon" role="presentation">
+            <use href="#icon-server-heart" xlink:href="#icon-server-heart"></use>
```

Comment:
AFAIK `xlink:href` is no longer needed:
https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/xlink:href
https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/href

I might be wrong though.